flow/src/main/java/com/sequenceiq/flow/core/Flow2Handler.java

```diff
@@ -332,13 +332,11 @@ private FlowAcceptResult createNewFlow(String key, Payload payload, FlowParamete
             runningFlows.put(flow, flowChainId);
             Benchmark.measure(() -> flowStatCache.put(flowId, flowChainId, payload.getResourceId(),
                     flowConfig.getFlowOperationType().name(), flow.getFlowConfigClass(), false), LOGGER, "Creating flow stat took {}ms");
-            transactionService.required(() -> {
```

Comment:
Do we really like to remove the transaction from here?

Reply:
you are right, it can cause unforeseen errors. I have reverted this change